apps/web/src/modules/create-proposal/components/UpgradeCard.tsx

```diff
@@ -82,7 +61,7 @@ export const UpgradeCard = ({
             borderRadius={'small'}
             className={image}
           >
-            {!!imgurl && !error && (
+            {!!imgurl && (
```

Comment:
nit: do we need this conditional anymore if the imgurl is a hardcoded value now?